www-apps/radicale/radicale-2.1.11-r2.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
```

Comment:
bump to EAPI 7

Reply:
563ebcb0cc1f4207ae53f70ac054ed73e687bbcf